### PR TITLE
Add EIP: Cease serving history before PoS

### PR DESCRIPTION
**Pull request by lightclient**
This EIP proposes that execution layer clients stop server historical data from before the merge over p2p.

### CODE REVIEW

File: EIPS/7639.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+eip: 7639
```
Comment (abcoathup) [inline EIPS/7639.md:2]: @lightclient need to rename the file to eip-7639.md

File: EIPS/7639.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+As of 2024, historical data in clients has grown to around 500 GB. Nearly 400 GB
+of that is from block data before PoS was activated in the Paris upgrade. Long
+term, Ethereum plans to bound the amount of data nodes must store. This EIP
+proposes the first steps to achieve such goal.
+
+## Specification
+
+Clients must not make or respond to p2p queries about blocks before block 15537393.
+
+### Header Accumulator
```
Comment (smartprogrammer93) [inline EIPS/7639.md:29]: This feels out of context. even though i know what this is, it does not seem connected to the rest of the EIP. maybe we need to add some explaination about how the header accumulator will be used. and how users will be able to get this data using era1 one files?

File: EIPS/7639.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+eip: 7639
+title: Cease serving history before PoS
+description: Execution layer clients will no longer serve block data before Paris over p2p.
+author: lightclient (@lightclient)
+discussions-to: https://ethereum-magicians.org/t/cease-serving-history-before-pos/18991
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-02-13
+---
```
Comment (smartprogrammer93) [inline EIPS/7639.md:11]: Maybe add dependency on eip-6110?

Comment (lightclient) [reply EIPS/7639.md]: I don't think this is dependent on 6110? This only removes pre-merge data. 6110 only modifies the behavior of exits after Prague, so clients still need to sort out how to deal with the loss of pre-merge receipts.

File: EIPS/7639.md (hunk 1 above)
Comment (lightclient) [reply EIPS/7639.md]: fixed, thx

File: EIPS/7639.md (hunk 2 above)
Comment (lightclient) [reply EIPS/7639.md]: I think you're right. I have moved it to a separate EIP: https://github.com/ethereum/EIPs/pull/8277

**Comment by lightclient** [comment]
mr. eip editor, my eip is ready

**Comment by erayack** [comment]
Do you see this as a step up to EIP-4444, or will it replace it?

**Comment by lightclient** [comment]
Hi @erayack this would be step up to full 4444.

File: EIPS/eip-7639.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+instead focuses on the shape of the data.
+
+## Backwards Compatibility
+
+After this EIP is activated, nodes will no longer be able to full sync from the
+devp2p network. To continue doing so, they must retrieve the data out-of-band.
+
+## Security Considerations
+
+TODO
```
Comment (SamWilsn) [inline EIPS/eip-7639.md:51]:
```suggestion
<!-- TODO -->
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**